src/pydolphinscheduler/examples/parameter_example.py

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+r"""
+A tutorial example take you to experience pydolphinscheduler.
+
+After tutorial.py file submit to Apache DolphinScheduler server a DAG would be create,
+and workflow DAG graph as below:
+
+                  --> task_child_one
+                /                    \
+task_parent -->                        -->  task_union
+                \                    /
+                  --> task_child_two
+
+it will instantiate and run all the task it have.
+"""
+
+# [start tutorial]
+# [start package_import]
+# Import Workflow object to define your workflow attributes
+from pydolphinscheduler.core.workflow import Workflow
+from pydolphinscheduler.core.parameter import ParameterType
+
+# Import task Shell object cause we would create some shell tasks later
+from pydolphinscheduler.tasks.shell import Shell
+from pprint import pprint
+
+# [end package_import]
+
+# [start workflow_declare]
+with Workflow(
+    name="tutorial",
+) as workflow:
+    task = Shell(name="task_parent",
+                 command="echo hello pydolphinscheduler",
+                 input_params={
+                     "value_VARCHAR": "abc",
+                     "value_LONG": ParameterType.LONG("1000000"),
```

Comment:
can we also find some way to auto-convert these parameter to java map?

Reply:
such as long, list, date?

Reply:
Yes, but I have no experience in using these parameters, so I don't know how to convert them